packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +56,22 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    frameOptions.iframeAttrs = frameOptions.iframeAttrs ? frameOptions.iframeAttrs : {};
+    if (Object.keys(frameOptions.iframeAttrs)?.includes('srcdoc')
+      && !Object.keys(frameOptions.iframeAttrs)?.includes('scrolling')) {
+      Object.assign(frameOptions.iframeAttrs, { scrolling: 'yes' });
+    }
+
+    frameOptions.resizeConfig = frameOptions.resizeConfig ? frameOptions.resizeConfig : {};
+    if (!Object.keys(frameOptions.resizeConfig)?.includes('scrolling')) {
+      Object.assign(frameOptions.resizeConfig, { scrolling: true });
+    }
+
```

Comment:
Do we still need this if this is only a documentation update for now?

Reply:
After checking in with @scottwilmarth on this, let me re-work this logic. We'll add a section about scrolling option to the Implementation Guide so that this change will be passive. The consumers will be responsible for setting the option.